tests/source/macros.rs

```diff
@@ -218,9 +219,9 @@ fn __bindgen_test_layout_HandleWithDtor_open0_int_close0_instantiation() {
 
 // #878
 macro_rules! try_opt {
-    ($expr:expr) => (match $expr {  
+    ($expr:expr) => (match $expr {
         Some(val) => val,
-          
```

Comment:
I think the trailing spaces in this test file should not be removed as removing it is probably part of the test.

Reply:
Oops! Editor trying to help out-- not intended.  Thanks for caching that >_<.